js/backup.js

```diff
@@ -769,10 +792,15 @@
   }
 
   Whisper.Backup = {
-    backupToDirectory: function() {
+    backupToDirectory: function(options) {
```

Comment:
Should we call this `exportToDirectory` for consistency with standalone app? Only if it’s not a breaking change.